pages/opam-using.html

```diff
@@ -59,14 +59,8 @@ <h2>Initializing opam</h2>
 <h2>Installing Coq</h2>
 
 <p>Depending on your operating system, installing Coq using opam
-may require you to first install some system packages. The recommended way to determine
-the names of required system packages is via the <code>opam-depext</code> tool:</p>
-
-<pre><code>opam install opam-depext
-opam-depext coq
-</code></pre>
-
-<p>For example, on Debian / Ubuntu, this command may list the <code>m4</code> system
+may require you to first install some system packages.
+For example, on Debian / Ubuntu, the following command may list the <code>m4</code> system
 package, which can then be installed as follows:</p>
 
 <pre><code>sudo apt-get install m4
```

Comment:
I don't like the new version. It says "the following command may list" but then it doesn't show the opam command but the `apt-get` command. Furthermore, IIUC, you don't actually need to run the `apt-get` command by yourself, opam 2.1 will do it for you.